### ISSUE TITLE
[a11y] Element does not have noticeable text that is visible to screen readers on the Program dashboard for Civiform Admin

### ISSUE THREAD

**Issue by plabanTest1** [civiform/civiform#9603]
**Describe the bug**
Element does not have inner text that is visible to screen readers
aria-label attribute does not exist or is empty

**To Reproduce**

1. Go to https://qa.civiform.dev/programs
2. Click on DevTools
3. Select Log in as: Civiform Admin 
4. Run the Voiceover to the element and notice the readable text for it.

**Expected behavior**
The buttons should have discernible text.

**Screenshots**

![Image](https://github.com/user-attachments/assets/c5e020f2-9bab-4309-9692-b11c9de8cbfa)

**Desktop (please complete the following information):**

- OS: [iOS]
- Browser [Chrome]
- Version [e.g. 131.0.6778.266]
- Accessibility Tools [Parashu]

**Additional context**

**Comment by nb1701** [comment]
Likely will change once we move to USWDS
